### ISSUE TITLE
LWC Retrieve from Sandbox not working

### ISSUE THREAD

**Issue by nishanpatel** [forcedotcom/salesforcedx-vscode#893]
### Summary

_I am using Pre-Release Salesforce extension pack and installed LWC plugin in my VS code. I have connected my VS code with one of the existing Sandbox. However, I am not able to retrieve my LWC already created in my Sandbox_.

_I have followed instruction mentioned in [Salesforce Development Guide](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.install_setup_develop)  but still not able to retrieve LWD components._ 

### Steps To Reproduce:

1.  Follow the instruction mentioned on above link.
1.  Connect VS code with Sandbox.
1.  Create LWC component using VS code.
1.  Create another project and connect above sandbox and retrieve same project with LWC.
1.  Check the lwc folder in VS code project.

### Expected result

_It should also retrieve created LWC metadata along with other metadata_.

### Actual result

_VS code is not retrieving LWD components along with other metadata_.

### Additional information

**VS Code Version**: 1.30.2

**SFDX CLI Version**: 45.0.12 (pre-release)

**OS and version**: Windows 10

**Comment by ntotten** [comment]
I am assuming that after step 3, you deployed the LWC component to the sandbox, right?

**Comment by nishanpatel** [comment]
@ntotten yes thats right. Firstly I have created LWC using VS code and then created new project or someone else retrieve using VS code, LWC is not getting into new project workspace.

**Comment by nishanpatel** [comment]
Hi @ntotten we have just verified same retrieval LWC is working fine in Mac and having issue only with windows machine.

**Comment by lcampos** [comment]
Closing as a duplicate of #1023 since that thread contains more details.
